cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/testdata/gcloud-util.sh

```diff
@@ -5,44 +5,89 @@
   fi
 
   local REGISTRY_NAME=$1
   local REGISTRY_FULL_PATH=projects/vmware-kubeapps-ci/locations/$FLUX_TEST_GCP_LOCATION/repositories/$REGISTRY_NAME
 
   echo
-  echo -e Checking if artifact registry [${L_YELLOW}$REGISTRY_NAME${NC}] exists...
+  echo -e Checking if artifact repository [${L_YELLOW}$REGISTRY_NAME${NC}] exists...
   RESP=$(gcloud artifacts repositories list --location=$FLUX_TEST_GCP_LOCATION --format "json" | jq --arg REGISTRY_FULL_PATH "$REGISTRY_FULL_PATH" '.[] | select(.name==$REGISTRY_FULL_PATH)')
   if [[ "$RESP" != "" ]] ; then
     echo -e Deleting repository [${L_YELLOW}$REGISTRY_NAME${NC}]...
     gcloud artifacts repositories delete $REGISTRY_NAME --location=$FLUX_TEST_GCP_LOCATION -q 
   fi
 }
 
-function createArtifactRegistry()
+function createGcpArtifactRepository()
 {
   # sanity check
   if [[ "$#" -lt 1 ]]; then
-    error_exit "Usage: createArtifactRegistry name"
+    error_exit "Usage: createGcpArtifactRepository name"
   fi
 
   local REGISTRY_NAME=$1
   local REGISTRY_FULL_PATH=projects/vmware-kubeapps-ci/locations/$FLUX_TEST_GCP_LOCATION/repositories/$REGISTRY_NAME
 
   echo
-  echo -e Checking if artifact registry [${L_YELLOW}$REGISTRY_NAME${NC}] exists...
+  echo -e Checking if artifact repository [${L_YELLOW}$REGISTRY_NAME${NC}] exists...
   RESP=$(gcloud artifacts repositories list --location=$FLUX_TEST_GCP_LOCATION --format "json" | jq --arg REGISTRY_FULL_PATH "$REGISTRY_FULL_PATH" '.[] | select(.name==$REGISTRY_FULL_PATH)')
   if [[ "$RESP" != "" ]] ; then
-       echo -e "Artifact registry [${L_YELLOW}${$REGISTRY_NAME}${NC}] already exists in harbor..."
+       echo -e "Artifact repository [${L_YELLOW}${$REGISTRY_NAME}${NC}] already exists in harbor..."
   else 
     gcloud artifacts repositories create $REGISTRY_NAME --repository-format=docker --location=$FLUX_TEST_GCP_LOCATION --description="Helm repository for kubeapps flux plugin integration testing"
   fi
 
   # configure Docker with a credential helper to authenticate with Artifact Registry
   gcloud auth configure-docker $FLUX_TEST_GCP_REGISTRY_DOMAIN
 }
+
+function setupGcrStefanProdanClone {
+  # this creates a clone of what was out on "oci://ghcr.io/stefanprodan/charts" as of Jul 28 2022
+  # to oci://demo.goharbor.io/stefanprodan-podinfo-clone
+  
+  # TODO commands below require
+  #  $ gcloud auth login
+  # to have run first and 
+  #  $ gcloud auth revoke 
+  # when done. In other words, my personal GCP login. Need to switch to use a service account 
+  
+  local REPOSITORY_NAME=stefanprodan-podinfo-clone
+  deleteGcpArtifactRepository $REPOSITORY_NAME
+  createGcpArtifactRepository $REPOSITORY_NAME
+
+  gcloud auth print-access-token | helm registry login -u oauth2accesstoken \
+    --password-stdin $FLUX_TEST_GCP_REGISTRY_DOMAIN
+  
+  trap '{
+    helm registry logout $FLUX_TEST_GCP_REGISTRY_DOMAIN
+  }' EXIT  
+
+  pushd $SCRIPTPATH/charts
+  trap '{
+    popd
+  }' EXIT  
+
+  SRC_URL_PREFIX=https://stefanprodan.github.io/podinfo
```

Comment:
Why relying on an external resource like this? Couldn't we store locally the 9 tgzs and push them? We already have some of them stored (6.1.5 and 6.1.6).
This way we wouldn't be depending on Stefan Prodan removing or changing the repo.
We have this URL in multiple script files.

Reply:
in practice these files have not changed since they've been added to the repo by stefan. He adds new chart versions but never deletes old ones. but regardless, I will make the change you suggest. Greg